main.cpp

```diff
@@ -74,23 +74,29 @@ int main()
 
     std::cout << "Number of channels: " << channelWrapper.pointerToChannelStrip->addChannel(4) << " Number of inputs: " << channelWrapper.pointerToChannelStrip->numInputs << std::endl;
 
-    channelWrapper.pointerToChannelStrip->func();
+    std::cout << "Number of channels: " << channelWrapper.pointerToChannelStrip->addChannel(4) << " Number of inputs: " << channelWrapper.pointerToChannelStrip->numInputs << std::endl;
```

Comment:
You need to continue to call `func()` the way you did before.
Move `func()` to the cpp file.
- [ ] revise